scripts/fish.js

```diff
@@ -33,153 +35,153 @@ export class Fish {
 }
 
 export class FishEgg extends Fish {
-    constructor(position={x: WIDTH / 2, y: 100}) {
+    constructor(position = { x: WIDTH / 2, y: defaultStartingPositionY }) {
         const name = "Fish Egg";
         const imagePath = "fishEgg.jpg";
         const radius = 10;
-        const body = Matter.Bodies.circle(position.x , position.y, radius, {
+        const body = Matter.Bodies.circle(position.x, position.y, radius, {
             restitution: 0.1, // bounciness
             friction: 0.00001, // friction against other objects
         });
-        const next = new Minnow;
+        const next = new Minnow();
         super(name, imagePath, radius, body, next);
     }
 }
 
 export class Minnow extends Fish {
-    constructor(position={x: WIDTH / 2, y: 100}) {
+    constructor(position = { x: WIDTH / 2, y: defaultStartingPositionY }) {
         const name = "Minnow";
         const imagePath = "minnow.jpg";
         const radius = 20;
-        const body = Matter.Bodies.circle(position.x , position.y, radius, {
+        const body = Matter.Bodies.circle(position.x, position.y, radius, {
             restitution: 0.1, // bounciness
             friction: 0.00001, // friction against other objects
         });
-        const next = new Clownfish;
+        const next = new Clownfish();
         super(name, imagePath, radius, body, next);
     }
 }
 
 export class Clownfish extends Fish {
-    constructor(position={x: WIDTH / 2, y: 100}) {
+    constructor(position = { x: WIDTH / 2, y: defaultStartingPositionY }) {
         const name = "Clownfish";
         const imagePath = "clownfish.jpg";
         const radius = 30;
-        const body = Matter.Bodies.circle(position.x , position.y, radius, {
+        const body = Matter.Bodies.circle(position.x, position.y, radius, {
             restitution: 0.1, // bounciness
             friction: 0.00001, // friction against other objects
         });
-        const next = new MoorishIdol;
+        const next = new MoorishIdol();
         super(name, imagePath, radius, body, next);
     }
 }
 
 export class MoorishIdol extends Fish {
-    constructor(position={x: WIDTH / 2, y: 100}) {
+    constructor(position = { x: WIDTH / 2, y: defaultStartingPositionY }) {
         const name = "Moorish Idol";
         const imagePath = "moorishIdol.jpg";
         const radius = 40;
-        const body = Matter.Bodies.circle(position.x , position.y, radius, {
+        const body = Matter.Bodies.circle(position.x, position.y, radius, {
             restitution: 0.1, // bounciness
             friction: 0.00001, // friction against other objects
         });
-        const next = new Otter;
+        const next = new Otter();
```

Comment:
Don't use these parentheses, we don't want the next variable to call the constructor, we just want it to point to the next object